### PR TITLE
Remember me

### PR DESCRIPTION
**Pull request by jtwiest**
Added UI support for the remember me field.

### CODE REVIEW

**Comment by jmealo** [comment]
:+1:

**Comment by jmealo** [comment]
@themightychris this looks safe to merge?

File: html-templates/subtemplates/forms.tpl [hunk 1]
```diff
@@ -69,3 +69,4 @@
 
 {template loginField}{field inputName=_LOGIN[username] label=Username required=true attribs='autofocus autocapitalize="none" autocorrect="off"' hint='You can also log in with your email address.'}{/template}
 {template passwordField}{field inputName=_LOGIN[password] label=Password hint='<a href="/register/recover">Forgot?</a>' required=true refill=false type=password}{/template}
+{template rememberMeField}{if $.cookies['username'] !== null}{$atts = 'CHECKED'}{/if}{checkbox _LOGIN[remember] '' attribs=$atts label='Remember Me'}{/template}
```
Comment (themightychris) [inline html-templates/subtemplates/forms.tpl]: @jtwiest use the more concise dwoo array syntax `$.cookies.username`

Also for the use of `{checkbox}` don't use unnamed parameters

**Comment by themightychris** [comment]
closing because "remember me" seems to be a bit of an anti-pattern anymore. Browsers provide this and major websites don't offer it anymore

**Pull request closed without merging**